src/gantry/screens.py

```
@@ -320,6 +321,105 @@ class ClusterScreen(Screen):
+        },
+        "Namespaces": {
+            "all": (["Name", "Status", "Age"], ["name", "status", "age"]),
+            "single": (["Name", "Status", "Age"], ["name", "status", "age"]),
+        },
+        "Nodes": {
+            "all": (["Name", "Status", "Roles", "Version"],
+                    ["name", "status", "roles", "version"]),
+            "single": (["Name", "Status", "Roles", "Version"],
+                       ["name", "status", "roles", "version"]),
+        },
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)